### PR TITLE
[topic page] Add climate equity charts

### PR DESCRIPTION
**Pull request by chejennifer**
- add climate equity scatter charts
- update topic pages to remove empty charts

North Carolina:
Screenshot 2023-06-07 at 10 01 51 AM
Screenshot 2023-06-07 at 10 02 00 AM

Wayne County:
Screenshot 2023-06-07 at 10 02 47 AM

### CODE REVIEW

**Review by pradh — COMMENTED** [review]
Nice!  Are the screenshots not lining up with the config?

**Review by pradh — APPROVED** [review]
Thanks for doing this!

File: server/config/topic_page/equity/US_Places.textproto [hunk 1]
```diff
@@ -35,6 +35,265 @@ metadata {
...
+    title: "Health"
+    columns {
+      tiles {
+        title: "Asthma Prevalence vs. CO2 Emissions"
+        type: SCATTER
+        stat_var_key: "asthma_prevalence"
+        stat_var_key: "CO2_emissions"
+      }
+      tiles {
+        title: "Diabetes Prevalence vs. Predicted Max Temperature"
```
[the remark this answers is not in the document]
Comment (chejennifer) [reply server/config/topic_page/equity/US_Places.textproto]: done

File: server/config/topic_page/equity/US_Places.textproto [hunk 2]
```diff
@@ -35,6 +35,265 @@ metadata {
...
+      tiles {
+        title: "Disabilities vs. Predicted Max Temperature"
+        type: SCATTER
+        stat_var_key: "disabilities_prevalence"
+        stat_var_key: "max_temp_prediction"
+      }
+    }
+    columns {
+      tiles {
+        title: "Cardiovascular Disease Prevalence vs. Predicted Max Temperature"
```
[the remark this answers is not in the document]
Comment (chejennifer) [reply server/config/topic_page/equity/US_Places.textproto]: updated

File: server/config/topic_page/equity/US_Places.textproto [hunk 3]
```diff
@@ -35,6 +35,265 @@ metadata {
...
+      stat_var: "Annual_Emissions_CarbonDioxide_NonBiogenic"
+      name: "CO2 Emissions"
+    }
+  }
+
+  stat_var_spec {
+    key: "max_temp_prediction"
+    value {
+      stat_var: "MaxTemp_Daily_Hist_5PctProb_Greater_Atleast1DayADecade_CMIP6_Ensemble_SSP585"
+      name: "MaxTemp Ensemble GaussianMixture 5PctProb Greater Atleast1DayAYear CMIP6 SSP245"
```
Comment (pradh) [inline server/config/topic_page/equity/US_Places.textproto:146]: Ohh, this is incorrect.  Lets do:

"Projected max temp (5% probability, decadal, SSP245)"

**Comment by pradh** [comment]
> > > > Nice! Are the screenshots not lining up with the config?
> > > 
> > > 
> > > how so?
> > 
> > 
> > I see Gaussian Mixture as the variable name, but the config has the histogram based one.
> 
> The variable name is set in the config as well .. so for the temperature variable, what is the stat var dcid and what is the name we should use for it?

Ah, I missed that.  That should be fixed then.

**Comment by pradh** [comment]
Couple of questions (no need for changes unless its really trivial :):

1. Is there a way to add place-type to title?  Right now its not obvious what the dots are.
2. I'm unable to hover over the dots, like on NL scatter plots. Is that ability something that's built in the app (nl vs. topic page) rather than something that the tile impl provides?

**Pull request merged**